Makefile

```diff
@@ -130,18 +134,36 @@ build-monitoring:
 	cd monitoring && make build
 
 .PHONY: test-e2e
-test-e2e:
-	test/docker_e2e.sh
+test-e2e: down-locally build-dss build-monitoring start-locally probe-locally collect-local-logs down-locally
 
 tag:
 	scripts/tag.sh $(UPSTREAM_OWNER)/dss/v$(VERSION)
 
+.PHONY: start-locally
 start-locally:
-	build/dev/run_locally.sh
+	build/dev/run_locally.sh up -d
+	build/dev/wait_for_local_dss.sh
+
+.PHONY: probe-locally
+probe-locally:
+	monitoring/prober/run_locally.sh
 
+.PHONY: collect-local-logs
+collect-local-logs:
+	docker logs core-service-for-testing 2> core-service-for-testing.log
```

Comment:
`build/dev/run_locally.sh` uses `docker-compose_dss.yaml` where core service is named `local-dss-core-service`.

For some reason it is breaking locally and not catched by the CI.